src/components/Product/ProductCard/ProductCard.tsx

```diff
@@ -61,7 +57,8 @@ export default function ProductCard({
       </Link>
       <div className={' absolute right-0 top-0'}>
         <ButtonHeart
-          active={isActive}
+          // active={isActive}
```

Comment:
nit: do we need this commented out code?

Reply:
Thank you! Deleted.